server/src/test/java/org/elasticsearch/transport/RemoteClusterSettingsTests.java

```diff
@@ -71,9 +71,29 @@ public void testSeedsDefault() {
         assertThat(REMOTE_CLUSTER_SEEDS.getConcreteSettingForNamespace(alias).get(Settings.EMPTY), emptyCollectionOf(String.class));
     }
 
```

Comment:
nit: can you test that at the setting is filtered too ? ... it is kinda pointless test, but it expresses a deliberate intent to make it filtered.

Reply:
Done